lib/index.js

```diff
@@ -246,6 +279,82 @@ export class Baker extends EventEmitter {
     }
   }
 
+  /**
+   * Prepares a function to process crosswalk image paths.
+   * @param {string} crosswalkPath - The base path for crosswalk images.
+   * @returns {Function} A function that maps over data entries to update image paths.
+   */
+  prepareCrosswalk(/** @type {string} */ crosswalkPath) {
```

Comment:
if `crosswalkPath` and `prepareCrosswalk` cannot both be defined at the same time, why is `crosswalkPath` being passed as an arg here?